diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/KdocCommentsWarnTest.kt

```diff
@@ -509,4 +509,15 @@ class KdocCommentsWarnTest : LintTestBase(::KdocComments) {
             LintError(5, 5, ruleId, "${KDOC_NO_CLASS_BODY_PROPERTIES_IN_HEADER.warnText()} /**...")
         )
     }
+
+    @Test
+    fun `shouldn't trigger kdoc top level on actual methods`() {
```

Comment:
Do we need similar logic for all actual declarations, e.g. classes?

Reply:
I think yes. I added more tests.